site/learn/expert-advice/dev-tools/uuid-generator.html

```diff
@@ -30,4 +31,27 @@
   </form>
 </div>
 
-<script type="text/javascript" src="/assets/js/UUIDGenerator-0.1.0.js" defer></script>
\ No newline at end of file
+<h3>UUID Frequently Asked Questions</h3>
+<h4>What is a UUID?</h4>
+<p>A UUID (Universally Unique Identifier) is a string of characters that is used to uniquely identify an object. You may also see the term GUID (globally unique identifier) which is also used. Both UUIDs and GUIDs are generated using random numbers, which makes it extremely unlikely that two of them will be the same.
```

Comment:
```suggestion
<p>A UUID (Universally Unique Identifier) is a string of characters that is used to uniquely identify an object. You may also see the term GUID (Globally Unique Identifier) which is also commonly used. Both UUIDs and GUIDs are generated according to a standard, using a combination of random numbers, timestamps and algorithms, all of which makes it extremely unlikely that two of them will be the same. 
```

Reply:
> A UUID (Universally Unique Identifier) is a string of characters that is used to uniquely identify an object.

This is fine - but this is saying what it is, and how FusionAuth uses them. A v4 UUID is simply a unique value in the form of `8-4-4-4-12` for a total length of 36. 

In practice, we (FusionAuth) and other developers use them as a primary key, or some other unique identifier of an object or thing.